### PR TITLE
quiz-questions

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Users can play quizzes
Users can quit the game
In the end, the final score would be displayed to the user
Users cannot skip a question

https://user-images.githubusercontent.com/37957073/160767638-fc2f4132-d783-4614-bd15-a3e055308d34.mp4

### CODE REVIEW

File: src/components/Questions/Questions.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  correct,
+  setScores,
+  scores,
+  setQuestions,
+}) => {
+  const [selected, setSelected] = useState();
+  const [error, setError] = useState(false);
+  const navigate = useNavigate();
+
+  const handleSelect = (i) => {
```
Comment (anupkgurung) [inline src/components/Questions/Questions.js]: suggestion :what is 'i' try to use some meaningful or specific name that could define its purpose.

File: src/components/Questions/Questions.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                {i}
+              </button>
+            ))}
+        </div>
+        <div className="controls">
+          <Button
+            variant="contained"
+            color="secondary"
+            size="large"
+            style={{ width: 185 }}
```
Comment (anupkgurung) [inline src/components/Questions/Questions.js:72]: try not to use inline styling

**Review by anupkgurung — APPROVED** [review]
Nice use of components.
Good of error handling component
left few suggestion.

**Review by maitrakhatri — APPROVED** [review]
Clean code.
Really liked the way of passing props its very clear to read and understand, I'll this this from now onwards. 
Just take care of NOT using inline styling

File: src/components/Questions/Questions.js (hunk 1 above)
Comment (ranitasahaa1230) [reply src/components/Questions/Questions.js]: Done

File: src/components/Questions/Questions.js (hunk 2 above)
Comment (ranitasahaa1230) [reply src/components/Questions/Questions.js:72]: Okay

**Comment by ranitasahaa1230** [comment]
> Clean code. Really liked the way of passing props its very clear to read and understand, I'll this this from now onwards. Just take care of NOT using inline styling

**Pull request merged**